install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -231,7 +231,7 @@ data:
 {{- end }}
 
 {{- if hasKey .Values.bpf "hostRouting" }}
-  enable-host-legacy-routing: {{ .Values.bpf.hostRouting | quote }}
+  enable-host-legacy-routing: {{ (not .Values.bpf.hostRouting) | quote }}
```

Comment:
The value description is:
```
  # -- Configure whether direct routing mode should route traffic via
  # host stack (true) or directly and more efficiently out of BPF (false) if
  # the kernel supports it. The latter has the implication that it will also
  # bypass netfilter in the host namespace.
  #hostRouting: true
```
Meaning, isn't the current setting correct here?

Reply:
Ah, you are right, I parsed the `bpf.hostRouting` flag as "should host routing mode be set to BPF" (i.e. the same output we get out of `cilium status`) and missed the actual documentation for that.
Will close the PR

Reply:
This other two commits in the PR would be useful though :-)